cdk/log_storage/__init__.py

```diff
@@ -0,0 +1,23 @@
+from aws_cdk import (
+    core,
+    aws_s3
+)
+
+class LogStorage(core.Stack):
+    def __init__(self, scope: core.Construct,
+                 stage: str, *, env: core.Environment):  
+              
+        super().__init__(scope, f'LogStorage-{stage}', env=env)
+
+
+        self.s3_log_bucket()
+
+
+        
+    def s3_log_bucket(self):
+        self.log_bucket = aws_s3.Bucket(self, 'quicksight-log-data',
+                        block_public_access=aws_s3.BlockPublicAccess.BLOCK_ALL,
+                        encryption=aws_s3.BucketEncryption.S3_MANAGED,
+                        versioned=True,
```

Comment:
What do you think about leaving versioning turned off? The Google Sheet they're using will already track edit history & I think this could cause the size of stored data to balloon pretty quickly if someone started running the google apps script on a high frequency (every 5 minutes for example).

Reply:
I agree that versioning is most likely not needed. Google sheets will keep backups of the log data if needed and I think we should avoid any potential problems with unnecessary data storage. 

Turned versioned=False, in new commit.